### ISSUE TITLE
Add Profile URL as an extra message field

### ISSUE THREAD

**Issue by pklapperich** [therealklanni/slack-invite-request#5]
I'd like to see the URL sent as raw text so the link can be clicked in the android app. Right now, the profile link is only clickable on the website.

~~I'd also like to see the Google+ URL changed to "https://plus.google.com/u/0/[USERID]" This would load the profile page using the primary Google account, but makes it easy to change the URL to "1" or "2", etc to switch to a load it from a different signed in account.~~

**Comment by therealklanni** [comment]
@pklapperich the URL comes from the Google+ API response, we're not manipulating the URL in any way, and there are probably reasons why doing so would be a bad idea (maybe the suggested URL format doesn't work for all users, which would result in broken links?).

As for including an extra field with just the profile URL, we can certainly do that.

**Comment by pklapperich** [comment]
All of the Google products that support multiple signin (Gmail, Google+, Drive, etc) support u/0/ in the url to load the page as the first account you signed in with, and it works even if you don't sign into multiple accounts in that session.

But I can respect that decision; they certainly could change their URL structure in the future. Truthfully the issue I have is something Google+ should fix; when I click the profile link it loads with my work account, and when I click up in the corner to switch accounts it takes me to my home screen (instead of reloading the profile page in the other account).

**Comment by therealklanni** [comment]
My point was not the `u/0/` part. I understand that that's used for identifying which authenticated account. But my point is that we can't assume they signed into SIR using their first (`u/0/`) account, so we have to let the response tell us which to use. Also I have seen some Google accounts have a slightly different URL (it may be accounts that don't use Google+, I don't know what the differentiator is). I'd rather err with caution.

**Comment by pklapperich** [comment]
Oh, I see. With regard to your first concern that doesn't matter. The u/0/ affects the person VIEWING the profile, not the person SUBMITTING the profile. Basically, when I add /u/1/ to a URL that says "View the given profile from the second account I logged in with" (usually my personal account). That lets me see if the profile I'm viewing is already in my circles.

With regard to the second, that might be a concern, and I can understand using caution. I know it works with profile urls in the following forms, with or without extra stuff on the end (ex: /posts)
https://plus.google.com/u/0/[USERID]
https://plus.google.com/u/0/+[FriendlyName]
https://plus.google.com/u/0?gpsrc=gplp0#[USERID]
https://plus.google.com/u/0/?gpsrc=gplp0#+[FriendlyName]
The latter two URL schemes seem to pop up on internet explorer, while browsing the same profiles on chrome/firefox gives the first two forms. I've actually copied a URL from the first form and had it re-direct to one of the latter forms when pasting into internet explorer... no idea what that's about.

It saves me almost no time to add u/1/ vs changing u/0/ to u/1/ and like you said, caution is probably best. I struck that part from my original request.

**Comment by therealklanni** [comment]
Yep, you absolutely right, I was thinking of the situation inverted. So let's just focus on the extra URL field in the Slack message for now, we can open another issue to discuss potentially changing the base URL, if you still want to consider it.
